src/components/LoginForm.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import NoAccountInformation from './NoAccountInformation';
+
+class LoginForm extends React.Component {
+    state = {
+        email: '',
+        password: '',
+    };
+
+    handleEmailChange = e => {
+        this.setState({
+            email: e.target.value,
+        });
+    };
+    handlePasswordChange = e => {
+        this.setState({
+            password: e.target.value,
+        });
+    };
+    handleLoginFormSubmit = e => {
+        e.preventDefault();
+        console.log(this.state.email);
+        console.log(this.state.password);
+    };
+    render() {
+        return (
+            <form
+                className="login-container__form"
+                onSubmit={this.handleLoginFormSubmit}
+            >
+                <h1 className="login-container-form__title">Zaloguj się</h1>
```

Comment:
It seems a bit inconsistent to have two BEM blocks (bases): `login-container`, `login-container__form` and later on: `login-container-form__label`, `*__input`, `*__button`. Would be better to have one base: `login-section`, `login-section__form`, `login-section__input`, `*__label`, `*__button` etc.